### PR TITLE
comparing activity based IN with P3 IN

### PR DESCRIPTION
**Pull request by amylu00**
## Content

- add example where activity based parameterizations are compared to P3 ice nucleation parameterizations
- add to documentation

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by amylu00** [comment]
Final product: 
Screenshot 2024-02-27 at 10 38 29 AM

File: parcel/Example_P3_vs_activitybased.jl [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+ts_het = TD.PhaseNonEquil_pTq(tps, p₀, T₀_het, q)
+ts_hom = TD.PhaseNonEquil_pTq(tps, p₀, T₀_hom, q)
+ρₐ_dep = TD.air_density(tps, ts_dep)
+ρₐ_het = TD.air_density(tps, ts_het)
+ρₐ_hom = TD.air_density(tps, ts_hom)
+Rₐ = TD.gas_constant_air(tps, q)
+eₛ_dep = TD.saturation_vapor_pressure(tps, T₀_dep, TD.Liquid())
+eₛ_het = TD.saturation_vapor_pressure(tps, T₀_het, TD.Liquid())
+eₛ_hom = TD.saturation_vapor_pressure(tps, T₀_hom, TD.Liquid())
+e = qᵥ * p₀ * R_v / Rₐ
```
Comment (trontrytel) [inline parcel/Example_P3_vs_activitybased.jl:47]: Could you replace this with the function defined in `ParcelCommon.jl`? https://github.com/CliMA/CloudMicrophysics.jl/blob/c53aef88084caac64350c96330382f687cbce786/parcel/ParcelCommon.jl#L7

The same for ksi and S_i defined below. This helps ensure we don't have any typos in the initial conditions. (or I guess if we do then it's the same typo for all...)

File: parcel/Example_P3_vs_activitybased.jl [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            aerosol = aerosol_none,
+            deposition = deposition,
+            deposition_growth = deposition_growth,
+            size_distribution = size_distribution,
+        )
+        # solve ODE
+        local sol = run_parcel(IC_dep, FT(0), t_max, params)
+        # Plot
+        MK.lines!(ax1, sol.t, S_i.(sol[3, :], (sol[1, :])), label = deposition, color = :blue) # saturation
+        MK.lines!(ax2, sol.t, sol[9, :] * 1e-6, color = :blue)                                # ICNC; NOTE: this is multipled by 1e-8 to fit in same plot as ABDINM
```
Comment (trontrytel) [inline parcel/Example_P3_vs_activitybased.jl:108]: Is the comment about multiplying by 1e-8 still relevant?

File: parcel/Example_P3_vs_activitybased.jl [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+                deposition_growth = deposition_growth,
+                size_distribution = size_distribution,
+            )
+            # solve ODE
+            local sol = run_parcel(IC_het, FT(0), t_max, params)
+            # Plot
+            if aerosol == CMP.DesertDust(FT)
+                line_color = :green
+                MK.lines!(ax4, sol.t, S_i.(sol[3, :], (sol[1, :])), label = heterogeneous, color = line_color) # saturation
+                MK.lines!(ax5, sol.t, sol[9, :] .* 1e-8, color = line_color)                                   # ICNC
```
Comment (trontrytel) [inline parcel/Example_P3_vs_activitybased.jl:172]: I think it's better to keep `1e-6` everywhere and switch to the log scale. Or keep the scale but just mention in the text that describes the plot that the values from this configuration are not zero by 2 orders of magnitude smaller than the rest.

**Review by trontrytel — APPROVED** [review]
LGTM! Left some minor comments. 

Glad to see it all come together and looking forward to the calibration setups

**Pull request merged**